.github/workflows/nodejs.yml

```diff
@@ -8,7 +8,9 @@ jobs:
             matrix:
                 node-version:
                 - 12.x
+                - 14.x
```

Comment:
Do we need to test both v12 and v14; can't we assume testing the oldest and latest (unstable) version should suffice?

(I'd wanna avoid burning coal unnecessarily.)